From 815b6426b97231f7d92d2358853c9bcae8288806 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Fri, 14 Sep 2018 17:58:09 +0800
Subject: [PATCH 4654/5242] MLK-19591 ARM64: dts: change the vdev-nums
 accordingly

commit  412cb894fe2493790117598573a9971909b44626 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the missing vdev-nums updates for cm41 on imx8qm
mek board.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Acked-by: Jason Liu <Jason.hui.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
index 0b324ec..72ccfc8 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dtsi
@@ -1110,7 +1110,7 @@
 	/*
 	 * 64K for one rpmsg instance:
 	 */
-	vdev-nums = <1>;
+	vdev-nums = <2>;
 	reg = <0x0 0x90100000 0x0 0x20000>;
 	status = "okay";
 };
-- 
1.7.9.5

